gql/compat.py

```diff
@@ -0,0 +1,10 @@
+"""Compatibility helpers to smooth over differences between Python versions."""
```

Comment:
I'm open to changes to this filename, or combining it with an existing file 🙂

Reply:
I prefer to keep it inside the client.py for now. We can still move it to another file if we need to use it elsewhere later.